asp.net-app-single.yaml

```diff
@@ -105,24 +105,27 @@ resources:
         str_replace:
           template: |
             #region Deployment of IIS, FTP and ASP .Net Site
-            Function Deploy-AspdotNet(){
+            Function Deploy-AspdotNet
+            {
                 [CmdletBinding()]
                 Param(
-                    [String]$DomainName = "example.com",
-                    [String]$FtpUserName = "ftpuser01",
-                    [String]$FtpPassword = "Passw0rd",
-                    [String]$Logfile = "C:\Windows\Temp\Deploy-AspdotNet.log"
+                    [String]$DomainName = 'example.com',
+                    [String]$FtpUserName = 'ftp_user',
+                    [String]$FtpPassword = 'Zzs1vnC4gS3lp7',
```

Comment:
I might just be reading things wrong, but are we really putting hard-coded passwords in these templates?